### DISCUSSION TITLE
I made a Batch file for this!

### DISCUSSION THREAD

**Discussion by ghost** [mariot/chan-downloader#10] [General]
The .bat file request the tread url. Just paste and download begins.

`
@echo off
echo [32m4chan Downloader[0m
echo [33mhttps://github.com/mariot/chan-downloader[0m
set /p URL=[33mPaste [37m(CTRL+V)[0m [32m4chan[0m [33mthread[0m [37m(https://boards.4chan.org/x/thread/xxxxxxxx)[0m [33mURL and press ENTER:[0m
start chan-downloader -c 1 -i 5 -l 360 -r 5 -o %USERPROFILE%\Downloads -t %URL%

`

**Comment by mariot** [comment]
Hi! I think the locale is off in this file. I can't see all characters. But this looks awesome!

**Reply by ghost** [reply]
This special characters change the text color in CMD.
Just put the .bat file in same folder and run.
![SS1](https://i.ibb.co/K58rcWQ/Captura-de-tela-2021-06-24-153921.png)
![SS2](https://i.ibb.co/H4SjV9m/Captura-de-tela-2021-06-24-154044.png)
![SS3](https://i.ibb.co/P1bCVVg/Captura-de-tela-2021-06-24-154153.png)

**Reply by ghost** [reply]
[Downloader.txt](https://github.com/mariot/chan-downloader/files/6711539/Downloader.txt)

**Reply by mariot** [reply]
Thank you so much for this!

**Reply by ghost** [reply]
Thank you for this code! I have used it frequently.
Consider enabling the thread name when saving!!! =)
